maintainers/maintainer-list.nix

```diff
@@ -3104,6 +3104,12 @@
     githubId = 2195834;
     name = "Jaakko Luttinen";
   };
+  jm2dev = {
+    email = "jomarcar@gmail.com";
+    github = "jm2dev";
+    githubId = 474643;
+    name = "José Miguel Martínez Carrasco";
+  };
```

Comment:
this should be a separate commit, look at https://github.com/NixOS/nixpkgs/pull/74389 as an example

Reply:
Done.

Reply:
please reverse the order of the commits:

```
git rebase -i HEAD^^
# switch the two lines
# save
git push --force ...
```

Reply:
otherwise checking out the package commit will fail to evaluate

Reply:
Output of `git log --oneline` after following your instructions (and pushing to gh):

```
0bced444e7f (HEAD -> pytest-bdd, origin/pytest-bdd) pythonPackages.pytest-bdd: init at 3.2.1
b1601d57ec9 maintainers: add jm2dev
```

though github web doesn't show this way.

Reply:
ah, I looked at your branch, and it still showed it the other way. guess it is just the web view